lib/foods.js

```diff
@@ -63,6 +61,22 @@ $('document').ready(function() {
     }
   })
 
+
+  // Calculate total and remaining calories for breakfast meal table
```

Comment:
@mollybrown Like we just talked about - I like how you want to do this in a different way.  So let's pull this out for now until we have ID's on the foods.